docker-compose.yml

```diff
@@ -12,8 +13,15 @@ services:
       - ./backend/db-setup/create-db.sh:/docker-entrypoint-initdb.d/01-user-db-creation.sh
       - ./backend/db-setup/create-metabase-db.sh:/docker-entrypoint-initdb.d/02-metabase-db-creation.sh
       - ./backend/db-setup/reset-db.sql:/usr/local/lib/reset-db.sql
+      - ./backend/db-setup/anonymize-detect.sh:/usr/local/lib/anonymize-detect.sh
+      - ./backend/db-setup/anonymize-db.sql:/usr/local/lib/anonymize-db.sql
+      - ./backend/db-setup/generate_db_data.py:/opt/db/generate_db_data.py
       - ./backend/db-setup/reset-metabase-db.sql:/usr/local/lib/reset-metabase-db.sql
-    command: -p ${SR_DB_PORT:-5432}
+      - ./backend/db-setup/restore-db.sh:/usr/local/lib/restore-db.sh
+      - ./backend/db-setup/snapshot-db.sh:/usr/local/lib/snapshot-db.sh
+      - ./backend/db-setup/.pgpass/:/usr/local/lib/.pgpass
+      - ./backend/db-setup/snapshots/:/usr/local/lib/snapshots/
+    # command: -p ${SR_DB_PORT:-5432}
```

Comment:
Is this a stray?

Reply:
Hmm, I don't remember commenting this line. I'll put this back and thoroughly test before pushing.

Reply:
I don't use the `SR_DB_PORT` variable, but removing this would cause issues for anybody using it. I'm confident that `5432` is the default anyway so that behavior wouldn't change. A stray, I believe it is!

Reply:
Uncommented this line and pushed!

Reply:
> I don't use the SR_DB_PORT variable, but removing this would cause issues for anybody using it

This would explain why my local DB setup blew up on this branch 🙂 thanks!